packages/main/src/SegmentedButton.ts

```diff
@@ -261,22 +255,16 @@ class SegmentedButton extends UI5Element {
 	/**
 	 * Currently selected item.
 	 *
-	 * @readonly
-	 * @type {sap.ui.webc.main.ISegmentedButtonItem}
```

Comment:
we remove interfaces only for abstract components (without template).

Reply:
Done